### DISCUSSION TITLE
Apache Kvrocks 毕业随感 | 夜天之书

### DISCUSSION THREAD

**Discussion by giscus** [tisonkun/blog#87] [Comments]
# Apache Kvrocks 毕业随感 | 夜天之书

Apache Kvrocks 是一个分布式 KV 数据库，使⽤ RocksDB 作为底层存储引擎并兼容 Redis 协议，旨在解决 Redis 内存成本⾼以及容量有限的问题，可以作为 Redis 的持久化变体做 drop-in 替换。 Kvrocks 起初是美图内部研发的软件，于 2019 年对外开源并独立运营。2022 年 4 月，Kvrocks 在 Champion 陈亮的推动下进入 ASF

https://www.tisonkun.org/2023/06/28/kvrocks-graduate

**Comment by ZoomQuiet** [comment]
#是也乎,(￣▽￣)
可以看出, 这一系列优秀的体验, 在一般企业中基本不可能实现:

1: 私人体验为核心的策略
2: 具体行为导向的信任跃迁
3: 先给权限再进行考察
4: 根据具体问题随时引入新工具流程成员...

没有 KPI, 没有 OKR 只有对具体技术问题的高速交流和推进...

嘦能配套可行的商业模式, 这种团队几乎总是完胜所有 KPI 驱动的...

可惜...一般并不能配套上.

**Comment by kimmking** [comment]
好文，每个proposal伊始时的mentor都应该读一读。
